### ISSUE TITLE
Latest update (4.4.0-1-02) reset settings and causing connection issues

### ISSUE THREAD

**Issue by mchamp00** [binhex/arch-qbittorrentvpn#121]
The latest image (4.4.0-1-02) seems to have broken my connection. I have the little orange whatever it is at the bottom instead of the green plug and I can't connect any peers or even the tracker since updating to this version. The change also reset many of the settings (e.g. reenabled DHT, random port, etc), but even after adjusting them to what they should be, I can't connect to anything. It also seems to respond a lot slower than prior versions. I went back to the prior image (4.3.9-2-01) and it's working fine again. Doubt this is an issue on my end unless I somehow missed a setting that changed.

**Comment by TheFeelTrain** [comment]
Similar thing happened to me. All of my settings were reset, although my connection is fine. I can't see any torrent information. The general, trackers, peers, and content tabs all do not load and appear blank even though the torrents themselves are still seeding.

**Comment by mchamp00** [comment]
I noticed that issue as well, even after I rolled back to the old image. However, when I switched from Chrome to Microsoft Edge (yuck), those tabs load again, so it might be a caching issue.

**Comment by TheFeelTrain** [comment]
It does seem to be the cache. I used "empty cache and hard reload" in Chrome and now the tabs load.

**Comment by Themis3000** [comment]
My default save location changed to /config/downloads for some reason. It started moving all my torrents to that location automatically from /data. Hopefully it didn't make me lose any data since /config/downloads wasn't mapped to be persistent on my docker image and it appeared the container crashed at some point

**Comment by wgstarks** [comment]
There is a bug in the current version of qbittorrent. You should revert to the previous version.

**Comment by Themis3000** [comment]
> There is a bug in the current version of qbittorrent. You should revert to the previous version.

Thanks, I revered and everything is fixed, no data lost. Hopefully the issues with 4.4.0 are fixed soon.

**Comment by gdreelin** [comment]
Yeah I had the same issue as Themis3000 it sent all the files to /config/download instead of /data like it is configured to do. I could see it downloading the files that were sent to it from Radarr and Sonarr but after they completed they just would not appear in the /data/Radarr or /data/Sonarr directory for transfer to their correct locations. I rolled back to last manual build for now.

**Comment by nwithan8** [comment]
Seconding @gdreelin , caused my cache drive to overflow because the download path inside the container got reset.

**Comment by gdreelin** [comment]
> Seconding @gdreelin , caused my cache drive to overflow because the download path inside the container got reset.

Yeah once I rolled back it was all good at that point. I had the same issue with one of the Plex Server updates breaking the use of it.

**Comment by gdreelin** [comment]
I have not seen an update if it was fixed yet.

**Comment by TheFeelTrain** [comment]
I tested it for science, 4.4.1 seems to be working fine now.

**Comment by Themis3000** [comment]
> I tested it for science, 4.4.1 seems to be working fine now.

Lol thank you for your scientific contribution. I appreciate that you tested it, I'll try updating now too and update if I have success too or hit any issues.

**Comment by TheFeelTrain** [comment]
Yeah I thought worst-case scenario I just downgrade it again lol. I don't notice anything out of the ordinary so I guess it's good now. I don't actually see anything about it in the patch notes but maybe it was just a subsection of a different bug.

**Comment by Themis3000** [comment]
I just upgraded and I've got no issues too, this seems mystically patched in 4.4.1 somehow. I've double checked at it looks like all my settings and torrents are just how I left them before migrating.

**Comment by Themis3000** [comment]
> I just upgraded and I've got no issues too, this seems mystically patched in 4.4.1 somehow. I've double checked at it looks like all my settings and torrents are just how I left them before migrating.

Actually, not sure how I missed this, but it looks like a lot of my settings where changed. None of the settings changed where destructive though. Many settings stayed the same, while many others didn't. It seems work the upgrade to me, but be sure to screenshot your settings before going through with the upgrade.

**Comment by jheyneman** [comment]
I just ran into this today after a power outage, not related to an update.  I haven't updated to the newest version because it hasn't been whitelisted yet.  

Power came back on, qBittorrent UI just returns "Unauthorized". Checked my config and it's back to defaults with the default user/pass.
